grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -2364,7 +2366,7 @@ class StockMovementService {
     }
 
     void createMissingShipmentItems(StockMovement stockMovement) {
-        Requisition requisition = stockMovement.requisition.refresh()
+        Requisition requisition = Requisition.get(stockMovement.id)
```

Comment:
@jmiranda I assume the `refresh()` (now the fetch) is done to "undo" any changes introduced to requisition object, but it needs to be confirmed by @awalkowiak 
Since code LGTM, I will merge this one, so that it could be tested before the end of the sprint, and we can discuss your doubts later if it sounds OK.